packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -162,6 +164,34 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       config.denominatorPriceFeed && (await _createMedianizerPriceFeed(config.denominatorPriceFeed));
 
     return new BasketSpreadPriceFeed(web3, logger, baselinePriceFeeds, experimentalPriceFeeds, denominatorPriceFeed);
+  } else if (config.type === "test") {
```

Comment:
The `test` and `invalid` types are used to test that the OO Keeper uses `CreatePriceFeed` correctly to create new price feeds.